app/src/main/java/org/fossasia/openevent/general/similar_events/SimilarEventsFragment.kt

```diff
@@ -0,0 +1,85 @@
+package org.fossasia.openevent.general.similar_events
+
+import android.arch.lifecycle.Observer
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.support.v7.widget.LinearLayoutManager
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import kotlinx.android.synthetic.main.fragment_similar_events.*
+import kotlinx.android.synthetic.main.fragment_similar_events.view.*
+import org.fossasia.openevent.general.R
+import org.fossasia.openevent.general.event.*
+import org.fossasia.openevent.general.utils.Utils
+import org.koin.android.architecture.ext.viewModel
+import timber.log.Timber
+
+class SimilarEventsFragment : Fragment() {
+    private val similarEventsRecyclerAdapter: EventsRecyclerAdapter = EventsRecyclerAdapter()
+    private val similarEventsViewModel by viewModel<SimilarEventsViewModel>()
+    private lateinit var rootView: View
+    private var EVENT_TOPIC_ID: String = "EVENT_TOPIC_ID"
+    private var eventTopicId: Long = -1
+    private lateinit var linearLayoutManager: LinearLayoutManager
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        similarEventsRecyclerAdapter.setEventLayout(EventUtils.SIMILAR_EVENTS)
```

Comment:
Here is another case

Reply:
Underscore is also not allowed, move the package in event package and rename to topic

Reply:
Changed package name :+1: